### DISCUSSION TITLE
This editor is fucking awesome!

### DISCUSSION THREAD

**Discussion by aka-bassman** [yoopta-editor/Yoopta-Editor#36] [General]
Hi, i'm making a simple article editing functionality, and searched whole block editor libraries.
Yopta is the best and code quality is also amazing.
I can't wait for the v2 release!!

**Comment by Darginec05** [comment]
Wow! 
Thank you for your feedback ❤️

V2 is coming soon 😍

**Comment by aka-bassman** [comment]
I just tried v-1.9.9 rc with plugins, it's insane!!
I wanna try to create a custom plugin that can highlight text with a comment. Is there any drafted guideline or docs?
It does not need to be formal, just a piece code is okay.

Thank you :)

**Reply by aka-bassman** [reply]
oh, i'm gonna try it with looking this part. Thanks :D
https://github.com/Darginec05/Yopta-Editor/blob/yopta-editor%40v2/packages/yoopta-paragraph/src/ui/Paragraph.tsx

**Comment by Darginec05** [comment]
Today I started write docs. 
I'll ping you when it will be ready :)

**Comment by maZahaca** [comment]
Hey @aka-bassman ! we've recently moved to v4 of the editor, can you check if it satisfies your use cases now?

**Comment by vidyakin** [comment]
Yeaahh, make version vor VUE, pleeeeaaaaase!

**Reply by Darginec05** [reply]
@vidyakin thanks for feedback. 
We have plans to make this editor framework agnostic!
